weave-js/src/components/PagePanelComponents/Home/Browse3/NotFoundPanel.tsx

```diff
@@ -0,0 +1,19 @@
+import {ErrorPanel} from '@wandb/weave/components/ErrorPanel';
+import React, {FC} from 'react';
+
+import {Button} from '../../../Button';
+import {useClosePeek} from './context';
+
+export const NotFoundPanel: FC<{title: string}> = ({title}) => {
+  const close = useClosePeek();
+  return (
+    <div style={{display: 'flex', flexDirection: 'column', height: '100%'}}>
+      <div style={{alignSelf: 'flex-end', margin: 10}}>
+        <Button icon="close" variant="ghost" onClick={close} />
```

Comment:
I guess it must work this way currently, but this button seems odd if you are not in a peek drawer.

Reply:
added explicit hiding of this button